src/cardano.h

```diff
@@ -15,25 +15,34 @@ STATIC_ASSERT(LOVELACE_MAX_SUPPLY < LOVELACE_INVALID, "bad LOVELACE_INVALID");
 #define POOL_KEY_HASH_LENGTH 28
 #define VRF_KEY_HASH_LENGTH 32
 #define TX_HASH_LENGTH 32
-#define METADATA_HASH_LENGTH 32
+#define AUX_DATA_HASH_LENGTH 32
+#define POOL_METADATA_HASH_LENGTH 32
+#define CATALYST_REGISTRATION_PAYLOAD_HASH_LENGTH 32
+#define ED25519_SIGNATURE_LENGTH 64
 
 #define MINTING_POLICY_ID_SIZE 28
 #define ASSET_NAME_SIZE_MAX 32
 
+#define REWARD_ACCOUNT_SIZE (1 + ADDRESS_KEY_HASH_LENGTH)
```

Comment:
the reward "address" is not a proper address in the sense you could send funds there - I see that as the main rationale behind calling it an "account" instead. It's also how it's called in the Ledger specs: https://hydra.iohk.io/build/6026331/download/1/ledger-spec.pdf (page 9)

Reply:
[Here](https://docs.cardano.org/en/latest/learn/types-addresses.html#reward-account-addresses) they call it "Reward account address". There's mentions of "reward address" also in the ledger spec.

It's used in functions called `constructRewardAddressFromKeyPath` and even in that function it's then assigned to a constant called `ADDRESS_LENGTH`. It's used as the size for a variable called `rewardAddress`. `rewardAddress` is also used by withdrawals. It's also derived from addressParameters.

I think it would be less questionable to call it address, but I see that `rewardAccount` is also used for `poolParams` so I'm not sure it could be renamed at this point.

Reply:
In cardano-node they have `RewardAcnt` which gets serialised into `StakeAddress` :smile: Daedalus seems to be using reward account.

Reply:
nevertheless, this is something this PR is not really touching, it's rather related to the refactors made in the operator app that have been extracted